From 73ae9cbc2c9d488d67c95f3472eb51c8773d404a Mon Sep 17 00:00:00 2001
From: Michael Forney <mforney@mforney.org>
Date: Thu, 26 Mar 2020 17:08:26 -0700
Subject: [PATCH] Disable multilib directory with --disable-multilib

---
 gcc/Makefile.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gcc/Makefile.in b/gcc/Makefile.in
index 5d05e8e0dc8..1ba5ab61844 100644
--- a/gcc/Makefile.in
+++ b/gcc/Makefile.in
@@ -2103,7 +2103,7 @@ libgcc.mvars: config.status Makefile specs xgcc$(exeext)
 multilib.h: s-mlib; @true
 s-mlib: $(srcdir)/genmultilib Makefile
 	if test @enable_multilib@ = yes \
-	   || test -n "$(MULTILIB_OSDIRNAMES)"; then \
+	   && test -n "$(MULTILIB_OSDIRNAMES)"; then \
 	  $(SHELL) $(srcdir)/genmultilib \
 	    "$(MULTILIB_OPTIONS)" \
 	    "$(MULTILIB_DIRNAMES)" \
-- 
2.28.0

